src/map/atcommand.c

```diff
@@ -4212,12 +4212,19 @@ ACMD(mount_peco)
 		return true;
 	}
 	if ((sd->job & MAPID_THIRDMASK) == MAPID_MECHANIC) {
+		int mtype = MADO_ROBOT;
+		if (!*message)
+			sscanf(message, "%d", &mtype);
+		if (mtype < MADO_ROBOT || mtype >= MADO_MAX) {
```

Comment:
what if they pass any number other than MADO_ROBOT and MADO_SUITE?
which the madogear types doesn't exists or undefined.

Reply:
same as previous comment.